language/functional_tests/tests/testsuite/builtins/create_account.mvir

```diff
@@ -1,22 +0,0 @@
-import 0x0.LibraAccount;
-import 0x0.LibraCoin;
-
-main() {
-    let addr: address;
-    let account_exists: bool;
-    let ten_coins: LibraCoin.T;
-    let account_exists_now: bool;
-
-    addr = 0x0111111111111111111111111111111111111011111111111111111111111110;
-    account_exists = LibraAccount.exists(copy(addr));
-    assert(!move(account_exists), 83);
-
-    ten_coins = LibraAccount.withdraw_from_sender(10);
-    create_account(copy(addr));
```

Comment:
Could we keep this test by using `LibraAccountcreate(copy(addr))` here instead?

Reply:
done and moved it to `libra_account` also I included the checks in the other tests all in this one